esp/src/eclwatch/QuerySetDetailsWidget.js

```diff
@@ -232,7 +232,7 @@ define([
                         tooltip += " " + newValue[i].Time;
                 }
                 this.graphsTab.set("tooltip", tooltip);
-            } else if (name === "ResourceURLCount" && newValue) {
+            } else if (name === "ResourceURLCount" && newValue - 1) {
                 this.widget._Resources.set("title", this.i18n.Resources + " (" + newValue + ")");
```

Comment:
Are you trying to adjust the resource count down by 1?  If so should you subtract 1 from this line also?